xDSCResourceDesigner.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.1.1.2'
+ModuleVersion = '1.1.1.3'
```

Comment:
Hi Dave,

First off, looks awesome. Glad to see we're already starting to pull in some of the best community forks. :) 

I know this is a pretty big commit you're doing here, but per our [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#creating-a-new-resource-in-an-existing-module) we actually recommend that you *don't* rev the version number when submitting a pull request. Reason being that we're trying to move to some more consistent semantic versioning and will bump it up whenever we deploy from our master branches to the Gallery.  

Thanks,
Joey